maven-extension/README.md

```diff
@@ -63,6 +63,7 @@ The Maven OpenTelemetry Extension supports a subset of the [OpenTelemetry auto c
 | otel.exporter.otlp.headers   | OTEL_EXPORTER_OTLP_HEADERS  | Key-value pairs separated by commas to pass as request headers on OTLP trace and metrics requests.        |
 | otel.exporter.otlp.timeout   | OTEL_EXPORTER_OTLP_TIMEOUT  | The maximum waiting time, in milliseconds, allowed to send each OTLP trace and metric batch. Default is `10000`.  |
 | otel.resource.attributes | OTEL_RESOURCE_ATTRIBUTES | Specify resource attributes in the following format: key1=val1,key2=val2,key3=val3 |
+| otel.maven.instrumentation.mojos.enabled | OTEL_MAVEN_INSTRUMENTATION_MOJOS_ENABLED | Disable the creation of spans for each mojo goal execution, `true` or `false`. Convenient to reduce the number of spans for large builds. Default value: `true` |
```

Comment:
The description confused me as it implies disabling, but based on the code I think it's meant to be enabling.

Is that correct?

Reply:
I think that the proposed code keeps mojo spans enabled by default (see [L171](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/108/files#diff-8625ba4b8e68496923fa886962656dc739e75194535a5d9a9c7a81e555fff294R171) and code snippet below).

I'm wondering if the naming convention of to disable instrumentations on the Otel Java Agent, `OTEL_INSTRUMENTATION_[NAME]_ENABLED` , is intuitive enough for our use case on the Otel Maven extension. 

**❓  Would it be more intuitive to suffix the config param by `_DISABLED` (ie `OTEL_MAVEN_INSTRUMENTATION_MOJOS_DISABLED`) than to follow the `_ENABLED` convention of the Otel Java Agent?**




```
    this.mojosInstrumentationEnabled =
        Boolean.parseBoolean(
            StringUtils.defaultIfBlank(mojosInstrumentationEnabledAsString, "true"));
```

Reply:
Following the convention of the instrumentation repo, for better or worse, is probably a good idea. I guess @kenfinnigan's pointing to the comment starting with `Disable the creation` which makes it seem like this flags disables when it's true. I think this would read fine

"Whether to create spans for each mojo goal execution"

Reply:
Thanks @anuraaga , I have improved the documentation based on your recommendation.